includes/roles-and-capabilities.php

```diff
@@ -354,6 +356,38 @@ function people_views( $views ) {
 	return $views;
 }
 
+/**
+ * Filters a Profile Owners's ability to edit their profile post.
+ *
+ * @since 0.1.3
+ *
+ * @param array   $allcaps All the capabilities of the user
+ * @param array   $caps    Required capabilities
+ * @param array   $args    [0] Requested capability
+ *                         [1] User ID
+ *                         [2] Associated object ID
+ * @param WP_User $user    The current WP_User object
+ *
+ * @return array
+ */
+function profile_ownership( $allcaps, $caps, $args, $user ) {
+	if ( 'edit_post' !== $args[0] ) {
```

Comment:
Would it also make sense to check the post type against `get_post( $args[2] )->post_type` before continuing?

Reply:
👍 Good idea!